src/ol_infrastructure/applications/mitopen/Pulumi.applications.mitlearn.QA.yaml

```diff
@@ -41,6 +41,7 @@ config:
     POSTHOG_API_HOST: "https://app.posthog.com"
     POSTHOG_TIMEOUT_MS: 1000
     POSTHOG_PROJECT_ID: "63497"
+    MITOL_NOTIFICATION_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
```

Comment:
This can just be added in `__main__.py` since it's the same regardless of environment.

Reply:
moved it to main